app/views/cases/cover_pages/_data_requests.html.slim

```diff
@@ -18,9 +18,9 @@
           - case_details.data_requests.includes([:contact]).decorate.order(date_requested: :asc).each do |data_request|
             tr
               td
-                = data_request.location
+                = data_request.data_request_area.decorate.location
```

Comment:
It probably makes sense to have the data_request decorator have a location method